test/utils/helpers.ts

```diff
@@ -19,6 +20,13 @@ export const mockGetPaymentData = (searchQuery: string, offset: number, limit: n
 		}
 	}
 
+	// Sort the results by the sortBy field
+	const keys = ['payee_name', 'part_postcode', 'town', 'county_council']
+  	if (keys.includes(sortBy)) {
+		results = results.sort((r1, r2) => r1[sortBy as keyof typeof object] > r2[sortBy as keyof typeof object] ? 1 : -1)
```

Comment:
Think it might be better to reference the dummyResults instead of object:  
`sortBy as keyof typeof dummyResults[0]`